### PR TITLE
[telegram]: Use fresh connection for commands

### PR DESCRIPTION
**Pull request by niteshbalusu11**
https://github.com/alexbosworth/balanceofsatoshis/issues/350

Uses a fresh lnd connection every time you run a command on telegram.

Files changed:
telegram/run_telegram_bot.js
telegram/start_telegram_bot.js

New File:
telegram/get_lnds.js

### CODE REVIEW

File: telegram/start_telegram_bot.js [hunk 1]
```diff
@@ -378,7 +384,7 @@ module.exports = (args, cbk) => {
           try {
             await handlePendingCommand({
               from: ctx.message.from.id,
-              id: connectedId,
```
Comment (alexbosworth) [inline telegram/start_telegram_bot.js:381]: should this get deleted

**Comment by alexbosworth** [comment]
I guess a tradeoff is losing support for pgp encrypted credentials, but that might even be already broken in a restart case

File: telegram/start_telegram_bot.js (hunk 1 above)
Comment (niteshbalusu11) [reply telegram/start_telegram_bot.js:381]: oops, accidental deletion.

**Comment by niteshbalusu11** [comment]
Oh yeah, I haven't thought about the encryption scenario.

**Comment by niteshbalusu11** [comment]
Maybe a --skip-encryption flag will use new connection else it will fall back to using initial connection?

**Comment by alexbosworth** [comment]
I think we could consider encryption already broken because connect_to_telegram will restart and call run_telegram_bot which would then go and want to decrypt, so that could be a separate issue

A better fix is probably storing the credentials in memory so that if you do a repeat call to authenticated_lnd and it already delivered the credentials one time, it redelivers the same that it did before and skips going to disk to get them

**Comment by niteshbalusu11** [comment]
Cool ok. You want to deal with it now or in a separate issue later?

**Comment by alexbosworth** [comment]
probably better to deal with it separately because it is preexisting

**Comment by niteshbalusu11** [comment]
Cool ok.

**Pull request merged**